docs/source/app_developers_guide/docker.rst

```diff
@@ -0,0 +1,459 @@
+********************
+Using Docker Compose 
+********************
+
+Overview
```

Comment:
Removed header. Did you mean just let the intro live under the document title (Using Sawtooth with Docker)? That is what I did, and I'll also do the same for the Ubuntu guide.